### DISCUSSION TITLE
How to specify `cleanlab[datalab]` in `pyproject.toml`?

### DISCUSSION THREAD

**Discussion by nishamuktewar** [cleanlab/cleanlab#736] [Q&A]
I need to install/use `cleanlab[datalab]` but need to include this dependency in `pyproject.toml`. How can I do it?
Something like this `cleanlab[datalab] = "2.4.0"` doesn't seem to work. Any ideas? Thanks.

**Comment by elisno** [comment]
Hi Nisha,

I assume you're using something like `poetry` for dependency management if you need to include the dependency in `pyproject.toml`.

I think you can just specify an `extras` key for the `cleanlab` dependency. Does something like this work?

```toml
cleanlab = {version = "2.4.0", extras = ["datalab"]}
```

---

While the `Datalab` feature can help identify issues in your dataset, if your ultimate goal is to address these issues, you might find our tool, Cleanlab Studio ([link](https://cleanlab.ai/)), beneficial.

**Reply by nishamuktewar** [reply]
Thank you! Yes, I should have mentioned that I was using `poetry` for dependency management.
